docs/source/examples/demo_ml_commons_integration.ipynb

```diff
@@ -191,21 +195,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 30,
    "id": "3bfe1532",
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Model was registered successfully. Model Id:  t0jp24kBHUgZTTdfDMmm\n",
-      "t0jp24kBHUgZTTdfDMmm\n"
+      "Model was registered successfully. Model Id:  7aF5sYgBZqn0fcHifav4\n"
      ]
     }
    ],
    "source": [
-    "model_id = ml_client.register_pretrained_model(model_name = \"huggingface/sentence-transformers/all-MiniLM-L12-v2\", model_version = \"1.0.1\", model_format = \"TORCH_SCRIPT\", deploy_model=False, wait_until_deployed=False)"
+    "\n",
+    "model_id = ml_client.register_pretrained_model(model_name = \"huggingface/sentence-transformers/all-MiniLM-L12-v2\", model_version = \"1.0.1\", model_format = \"TORCH_SCRIPT\", model_group_id = \"d4hfsYgBFp6IJxCcqpwi\", deploy_model=False, wait_until_deployed=False)"
```

Comment:
Can we add some documentation reference here for creating model group id. Otherwise customer might get confused about this.

Reply:
I just removed this file. There is one PR that is already merged last week for correcting this notebook. So, the notebook in this PR is outdated. Sorry for confusion.